Measure the processing time of each smoke test and record the result in debug_messages

ykitamura-mdsol opened the pull request:
Record the processing time in `debug_messages`

It will show up like this on rake task:
<img width="218" alt="screen shot 2016-11-04 at 4 05 30 pm" src="https://cloud.githubusercontent.com/assets/16286071/19997274/a060e64e-a2a8-11e6-9b3a-f33e089bf070.png">

@jfeltesse-mdsol @jcarres-mdsol @cabbott @ssteeg-mdsol

JordiPolo commented:
I think this is worth a bigger update. 
`duration` or `processing_time` , (we need a native) can be a method in `Result` also in `EvaluationCollection` then they can be top level elements in the json output, console output looks good

JordiPolo commented:
Also would need to update docs. This time is in `ms` ?

ykitamura-mdsol commented:
The time is in `second` now, but it might be good to convert in `ms`

jfeltesse-mdsol commented:
LGTM

ykitamura-mdsol commented:
addressed comments

added `ms` on rake output
<img width="498" alt="screen shot 2016-11-07 at 5 37 23 pm" src="https://cloud.githubusercontent.com/assets/16286071/20052793/ee5b8d0a-a518-11e6-9c81-a8b565c45e38.png">

ssteeg-mdsol commented:
Change looks good - `duration` works well. 👍

jcarres-mdsol commented on lib/kapnismology/result.rb:
```diff
@@ -38,6 +42,10 @@ def format_debug_messages(debug_messages)
     end
+
+    def format_duration(duration)
+      duration ? "duration: #{Terminal.bold(duration)} ms\n" : ''
```
is possible for duration to not be set?

jcarres-mdsol replied:
Maybe should be initialized to 0 in all the result classes

ykitamura-mdsol commented:
addressed comments
